app/src/main/kotlin/co/netguru/baby/monitor/client/common/view/PulsatingView.kt

```diff
@@ -34,22 +34,22 @@ class PulsatingView(context: Context, attrs: AttributeSet?) : View(context, attr
 
     init {
         context.obtainStyledAttributes(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/85cb1800bcef802a6706010d0e78f2363392f785/app/src/main/kotlin/co/netguru/baby/monitor/client/common/view/PulsatingView.kt#L36"></span>This <code>TypedArray</code> should be recycled after use with <code>#recycle()</code></td>
    </tr>
  </tbody>
</table>